Bottom: 191b8a5b14c9c02c2a792002656d13d92b6885ee
Top:    109cd28ba86379f860c7f454d642364fe82f51dd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-15 01:46:42 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
